lms/djangoapps/grades/migrations/0001_initial.py

```diff
@@ -2,19 +2,42 @@
 from __future__ import unicode_literals
```

Comment:
The use of the table is behind a feature flag, the existence of previous migrations in migration histories is not.

I just tested switching from a fully migrated release devstack to this branch. `./manage.py migrate ...` does not find any changes, since migration 0001 has already been applied previously. So the new column isn't added at all, and the database state is no longer in sync with migration files.